exposed-jodatime/src/main/kotlin/org/jetbrains/exposed/sql/jodatime/DateColumnType.kt

```diff
@@ -236,17 +254,29 @@ class DateTimeWithTimeZoneColumnType : ColumnType<DateTime>(), IDateColumnType {
     override fun sqlType(): String = currentDialect.dataTypeProvider.timestampWithTimeZoneType()
 
     override fun nonNullValueToString(value: DateTime): String = when (currentDialect) {
-        is SQLiteDialect -> "'${SQLITE_AND_MYSQL_DATE_TIME_WITH_TIME_ZONE_FORMATTER.print(value)}'"
+        is SQLiteDialect -> {
+            val instant = java.time.Instant.ofEpochMilli(value.millis)
```

Comment:
Am I right that java.time.Instant is used here due to optionals in the pattern which are not supported by joda time?

In general, it looks like it would be better not to mix different datetime modules (even if one of them is part ot kotlin). But if there is no way to handle these optional parts of the datetime sting it's fine.

Reply:
> Am I right that java.time.Instant is used here due to optionals in the pattern which are not supported by joda time?

That's correct.